elfpy/utils/apeworx_integrations.py

```diff
@@ -593,14 +597,14 @@ def get_pool_state(txn_receipt: ReceiptAPI, hyperdrive_contract: ContractInstanc
     return PoolState(**hyper_dict)
 
 
-def _snake_to_camel(_snake):
+def snake_to_camel(_snake):
     """Convert snake_case to camelCase."""
     return "".join(word.capitalize() for word in _snake.split("_"))
 
 
-def _camel_to_snake(_camel):
-    """Convert camelCase to snake_case."""
-    return _camel.lower().replace(" ", "_")
+def camel_to_snake(camel_string: str) -> str:
```

Comment:
this should be a separate PR, I already have one up for this:
https://github.com/delvtech/elf-simulations/pull/537